tests/common/test_typing.py

```diff
@@ -310,3 +314,17 @@ def test_add_value_to_literal() -> None:
     add_value_to_literal(TestSingleLiteral, "green")
     add_value_to_literal(TestSingleLiteral, "blue")
     assert get_args(TestSingleLiteral) == ("red", "green", "blue")
+
+
+def test_get_generic_type_argument_from_instance() -> None:
+    # generic contains hint
+    instance = SimpleNamespace(__orig_class__=Incremental[str])
```

Comment:
we are trying to keep the common module as the upstream dependency of all other top level modules. so it is better to not import from `extract` here. You could just use any standard generic type for testing

Reply:
Great point - updated in a4e5c11